config/notifiers.go

```diff
@@ -142,6 +143,7 @@ type EmailConfig struct {
 	AuthIdentity string            `yaml:"auth_identity,omitempty" json:"auth_identity,omitempty"`
 	Headers      map[string]string `yaml:"headers,omitempty" json:"headers,omitempty"`
 	HTML         string            `yaml:"html,omitempty" json:"html,omitempty"`
+	TEXT         string            `yaml:"text,omitempty" json:"text,omitempty"`
```

Comment:
I think the key here is that `HTML` is all caps since it it an initialism, but `TEXT` would be written as `Text` since it's just a word.

Does that make sense?
https://github.com/golang/go/wiki/CodeReviewComments#initialisms gives move info on this case